SUMO-251500: Adding monitor's information to OTEL Apps Set2 (#4876)
SumoLogic/sumologic-documentation: docs/integrations/databases/opentelemetry/mariadb-opentelemetry.md

- Examine slow query trends to determine if there are periodic performance bottlenecks in your database clusters.

<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/MariaDB-OpenTelemetry/MariaDB-Slow-Queries.png' alt="Slow Queries" />

## Create monitors for MariaDB app

import CreateMonitors from '../../../reuse/apps/create-monitors.md';

<CreateMonitors/>

### MariaDB alerts

> | Name | Description | Alert Condition | Recover Condition |

Reviewer: ```suggestion
| Name | Description | Trigger Type (Critical / Warning / MissingData) | Alert Condition |
```

Reviewer 2: @amee-sumo We do not want the "Trigger Type (Critical / Warning / MissingData)" column here, as we are only showing the Alert and Recover conditions here.

|:--|:--|:--|:--|
| `MariaDB - Critical Errors` | This alert is triggered when there are critical database errors. | Count `>` 10 | Count `<=` 10 |
| `MariaDB - Excessive Slow Query Detected` | This alert is triggered when the average time to execute a query is more than 15 seconds for a 5 minute time interval. | Count `>=` 1 | Count `<` 1 |
| `MariaDB - Failed Login Attempts` | This alert is triggered when there are excessive failed login attempts in a short period. | Count `>=` 1 | Count `<` 1 |

> | `MariaDB - Instance down` | This alert is triggered when we detect that a MariaDB instance is down | Count `>=` 1 | Count `<` 1 |

Reviewer 2: ```suggestion
| `MariaDB - Instance down` | This alert is triggered when the MariaDB instance is down. | Count `>=` 1 | Count `<` 1 |
```
